atc/metric/emitter/prometheus.go

```diff
@@ -213,6 +214,17 @@ func (config *PrometheusConfig) NewEmitter(attributes map[string]string) (metric
 	})
 	prometheus.MustRegister(jobsScheduling)
 
+	jobsSchedulingDuration := prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace:   "concourse",
+		Subsystem:   "jobs",
+		Name:        "schedulingDuration",
+		Help:        "Duration of jobs being scheduled in milliseconds",
+		ConstLabels: attributes,
+		Buckets:     []float64{1, 60, 180, 300, 600, 900, 1200, 1800, 2700, 3600, 7200, 18000, 36000},
```

Comment:
I think the first bucket could be 30? I don't think it will schedule a job in 1ms in anyway.